Food.h

```diff
@@ -0,0 +1,28 @@
+#ifndef FOOD_H
+#define FOOD_H
+
+#include "Random.h"
+
+class Food
+{
+public:
+    Food(int x = 5, int y = 5)
+        :m_posX(x), m_posY(y)
```

Comment:
consider to use user defined class such as coordinates instead of two separate variables x and y